src/crimson/os/seastore/extentmap_manager/btree/extentmap_btree_node.h

```diff
@@ -0,0 +1,93 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+
+#pragma once
+
+#include "crimson/common/log.h"
+#include "crimson/os/seastore/seastore_types.h"
+#include "crimson/os/seastore/transaction_manager.h"
+#include "crimson/os/seastore/extentmap_manager.h"
+
+namespace crimson::os::seastore::extentmap_manager{
+
+struct ExtMapNode : LogicalCachedExtent {
+  TransactionManager* tm = nullptr;
+  using ExtMapNodeRef = TCachedExtentRef<ExtMapNode>;
+  depth_t depth = 0;
+
+  ExtMapNode(ceph::bufferptr &&ptr) : LogicalCachedExtent(std::move(ptr)) {}
+  ExtMapNode(const ExtMapNode &other)
+  : LogicalCachedExtent(other),
+    tm(other.tm),
+    depth(other.depth) {}
+
+  void set_depth(depth_t _depth) { depth = _depth; }
+  void set_tm(TransactionManager* _tm) { tm = _tm; }
+
+  LogicalCachedExtentRef make_duplicate(Transaction& t,
+	  LogicalCachedExtentRef ref) {
+    return tm->get_mutable_extent(t, ref);
+  };
+
+  using seek_lextent_ertr = ExtentMapManager::seek_lextent_ertr;
```

Comment:
seek_lextent implies moving a pointer/iterator/descriptor.  Rename to find_lextents.